component/searchBar.jsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export function SearchBar({ onSubmit }) {
+  const [search, setSearch] = useState('');
+
+  return (
+    <StyledSearchbar>
+      <input
+        placeholder="Search Book"
+        type="text"
+        value={search.toUpperCase()}
```

Comment:
Just my personal preference: Is it necessary to use `toUpperCase`? From a user point of view it could be a bit confusing.

Reply:
i deleted the searchbar component for this branch